### PR TITLE
update OLM subscription names

### PR DESCRIPTION
**Pull request by mfrancisc**
Update subscription name.

Depends on: https://github.com/codeready-toolchain/toolchain-cicd/pull/120

### CODE REVIEW

**Comment by mfrancisc** [comment]
/retest

updated cicd pr

**Comment by mfrancisc** [comment]
/retest

updated cicd scripts

**Review by MatousJobanek — APPROVED** [review]

File: make/run-cicd-script.mk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
-GH_OWNER=codeready-toolchain
+GH_OWNER=mfrancisc
```
Comment (xcoulon) [inline make/run-cicd-script.mk:2]: is it intended?

Comment (mfrancisc) [reply make/run-cicd-script.mk:2]: yes, this PR depends on the changes from: https://github.com/codeready-toolchain/toolchain-cicd/pull/120

Comment (mfrancisc) [reply make/run-cicd-script.mk:2]: once verified that tests are passing, I can merge the cicd changes and then update this PR and merge it.

**Review by xcoulon — APPROVED** [review]

File: make/run-cicd-script.mk (hunk 1 above)
Comment (mfrancisc) [reply make/run-cicd-script.mk:2]: makes sense, I usually add reminders but I forgot to do that here. I've clarified that in the description hopefully. Sorry for the confusion and thanks for the review.

Comment (ranakan19) [reply make/run-cicd-script.mk:2]: Thanks for the added description @mfrancisc 
and thanks for the suggestion @xcoulon! I had the same question, and although I am looking after the description change, I found the file changes first and this conversation was helpful!

**Review by ranakan19 — APPROVED** [review]

**Pull request merged**